### PR TITLE
test: Testing Report

### PR DESCRIPTION
**Pull request by thogarty**
- Created IBM test without external provider and updated UAT test to use it instead of the main example
- Updated UAT GHA testing workflow to output test report after test runs
- Update Prod GHA testing workflows to output test report after test runs

### CODE REVIEW

**Comment by d-bhola** [comment]
@thogarty Please share the final testing report (screenshot or logs) generated with this code.

**Comment by thogarty** [comment]
@d-bhola, we decided not to do screenshots (not easily viewable) and just have the reviewer look at the latest action runs for the PR.

You can view them here:
https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-fcr-prod-suite.yaml
https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-port-prod-suite.yaml
https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-metal-nimf-suite.yaml
https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-virtualdevice-prod-suite.yaml
https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-prod-suite-all.yaml
https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-uat-suite.yaml

**Comment by d-bhola** [comment]
> @d-bhola, we decided not to do screenshots (not easily viewable) and just have the reviewer look at the latest action runs for the PR.
> 
> You can view them here: https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-fcr-prod-suite.yaml https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-port-prod-suite.yaml https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-metal-nimf-suite.yaml https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-virtualdevice-prod-suite.yaml https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-prod-suite-all.yaml https://github.com/equinix/terraform-equinix-fabric/actions/workflows/terratests-uat-suite.yaml
👍

**Review by srushti-patl — COMMENTED** [review]
We haven't added README.md and terraform.tfvars.example files to exmples-without-provider folder.

**Comment by thogarty** [comment]
@srushti-patl , we aren't exposing those examples publicly. They're just for internal testing and don't require the additional context. We only need README.md's and .tfvars.example for published example modules

**Comment by d-bhola** [comment]
Is there a way to remove/modify the question marks for better presentation of the test report?

**Comment by thogarty** [comment]
@d-bhola , beyond the scope of this task. We're using third party tools for the heavy lifting and that would require a second fork (we're already using a fork of go-junit-report). If we had more control over the source code I'd be happy to oblige but it's too much effort for too little reward.

**Comment by d-bhola** [comment]
> @d-bhola , beyond the scope of this task. We're using third party tools for the heavy lifting and that would require a second fork (we're already using a fork of go-junit-report). If we had more control over the source code I'd be happy to oblige but it's too much effort for too little reward.

Ok.👍

**Review by d-bhola — APPROVED** [review]
Added few comments. Rest all looks good.

**Comment by d-bhola** [comment]
Needs rebase

File: tests/examples-without-external-providers/port-2-ibm2-connection/README.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
+# Fabric Port to Fabric IBM 2.0 Service Profile Connection
```
Comment (d-bhola) [inline tests/examples-without-external-providers/port-2-ibm2-connection/README.md:1]: @thogarty - As you mentioned in your previous comment, we don't need README.md here

File: tests/examples-without-external-providers/port-2-ibm2-connection/terraform.tfvars.example [hunk 2]
```diff
@@ -0,0 +1,25 @@
+equinix_client_id      = "MyEquinixClientId"
```
Comment (d-bhola) [inline tests/examples-without-external-providers/port-2-ibm2-connection/terraform.tfvars.example:1]: Same for this. We don't need `terraform.tfvars.example`

**Pull request merged**